benchmarking/streamlit/streamlit_utils.py

```diff
@@ -0,0 +1,84 @@
+from matplotlib.axes._axes import Axes
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from typing import List
+
+
+def plot_dataframe_summary(df_req_info, ax):
```

Comment:
Moved these functions into a shared utils file so they weren't repeated in each page.